src/mon/MDSMonitor.cc

```diff
@@ -1447,6 +1462,9 @@ int MDSMonitor::filesystem_command(
     if (r < 0 && r == -EAGAIN) {
       mon->osdmon()->wait_for_writeable(op, new C_RetryMessage(this, op));
       return -EAGAIN; // don't propose yet; wait for message to be retried
+    } else if (r == 0) {
+      mon->clog->info() << "MDS daemon " << who << " marked failed by "
+                        << op->get_session()->entity_name;
```

Comment:
"...by admin at <addr>"?  not sure client.1234 is that informative, but the IP would be.